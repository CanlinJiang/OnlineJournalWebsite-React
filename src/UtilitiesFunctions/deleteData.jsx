
export default function deleteData(urlWithId) {

    fetch(urlWithId, {
            method: "DELETE"
        }).then(() => {
            
        })

    return;
}
