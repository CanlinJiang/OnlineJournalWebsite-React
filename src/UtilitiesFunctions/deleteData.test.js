import deleteData from "./deleteData"

describe("deleteData tests", () => {
    it("renders correctly", () => {
       const url = global.url + "1"
        
      const { container } = render(deleteData(url));
      expect(container).toMatchSnapshot();
    });
  
  });