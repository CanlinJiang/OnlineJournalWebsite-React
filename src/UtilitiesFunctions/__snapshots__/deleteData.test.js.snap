// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deleteData tests renders correctly 1`] = `<div />`;
